src_plugins/show/ppx_deriving_show.ml

```diff
@@ -23,6 +23,7 @@ let attr_printer context = Attribute.declare "deriving.show.printer" context
   Ast_pattern.(single_expr_payload __) (fun e -> e)
 let ct_attr_printer = attr_printer Attribute.Context.core_type
 let constr_attr_printer = attr_printer Attribute.Context.constructor_declaration
+let rtag_attr_printer = attr_printer Attribute.Context.rtag
```

Comment:
I was lost for a while as to what `rtag` means. The answer is that this obscure name comes from the parsetree, and it means "a constructor declaration in a polymorphic variant type" (`r` means `row` and `tag` is supposed to suggest `constructor`, I guess). The other PR #267 uses `attr_printer field.prf_attributes` directly, and I suppose that usig the ppxlib mechanism as in here is better.